iosapp/lib-ios/Sources/LibApp/IOSReducer+BtnFallback.swift

```diff
@@ -0,0 +1,206 @@
+typealias OnboardingBtn = IOSReducer.Action.Interactive.OnboardingBtn
```

Comment:
this whole file exists because we have 3 buttons that we fire off from many onboarding screens, some screens have just one, some two, some all three. because of that, there's no real type-safe way i could figure out to prove i had handled every combination, because some of the combinations should never be possible. so i decided to have this concept of a emergency fallback "next screen".  i have something similar to this in the mac app onboarding too, and in production i log if i ever have to use one, and in debug/test builds it actually fatal errors before it gets here. but i think this is worth having, helps me sleep at night and might smooth over a bug temporarily.